tests/golden/data/output/issue-148.lua

```diff
@@ -1,5 +1,19 @@
 local normalSounds = {Sound("honk1.wav"), Sound("honk2.wav"), Sound("honk3.wav"), Sound("honk4.wav")}
-render.DrawBeam(self.StartPos - norm, self.EndPos, 64 * (1 - self.Life), texcoord, texcoord + self.Length / 128, Color(Lerp(self.Life, 255, 50), Lerp(self.Life, 100, 50), 255, 255)) -- Start -- End -- Width -- Start tex coord -- End tex coord -- Color (optional)
+render.DrawBeam(
+    self.StartPos - norm, -- Start
+    self.EndPos, -- End
+    64 * (1 - self.Life), -- Width
+    texcoord, -- Start tex coord
```

Comment:
I'm really happy with this change. The comments used to all be collected and printed at the end. Now they're printed along with the argument. Besides that, having comments in function calls causes the function to be considered multiline.